doc/install.rst

```diff
@@ -8,33 +8,26 @@ Install third-party libraries
 
 The following third party libraries (TPLs) are used by mfmg:
 
-+------------------------+-----------------------------------+
-| Packages               | Version                           |
-+========================+===================================+
-| ARPACK                 | N/A                               |
-+------------------------+-----------------------------------+
-| Boost                  | 1.65.1                            |
-+------------------------+-----------------------------------+
-| BLAS/LAPACK            | N/A                               |
-+------------------------+-----------------------------------+
-| deal.II                | 9.0                               |
-+------------------------+-----------------------------------+
-| MPI                    | N/A                               |
-+------------------------+-----------------------------------+
-| Trilinos               | 12.X (master for CUDA support)    |
-+------------------------+-----------------------------------+
++------------------------+------------------------+
+| Packages               | Version                |
++========================+========================+
+| ARPACK                 | N/A                    |
++------------------------+------------------------+
+| Boost                  | 1.65.1                 |
++------------------------+------------------------+
+| BLAS/LAPACK            | N/A                    |
++------------------------+------------------------+
+| deal.II                | development            |
++------------------------+------------------------+
+| MPI                    | N/A                    |
++------------------------+------------------------+
+| Trilinos               | 12.X                   |
```

Comment:
`12.12.1`?

Reply:
We only use `Epetra` and `EpetraExt` which haven't changed in ages so any version of Trilinos should be good.